cmd/skupper/skupper.go

```diff
@@ -440,10 +441,26 @@ installation that can then be connected to other skupper installations`,
 				return fmt.Errorf("The --enable-flow-collector option must be used with the --enable-console option")
 			}
 
+			if routerCreateOpts.AuthMode != "internal" && (len(routerCreateOpts.User) > 0 || len(routerCreateOpts.Password) > 0) {
+				return fmt.Errorf("the --console-auth option must be set to internal")
+			}
+
+			if routerCreateOpts.AuthMode == "internal" && (len(routerCreateOpts.User) == 0 || len(routerCreateOpts.Password) == 0) {
+				return fmt.Errorf("the user and password values need to be configured if the --console-auth is set to internal")
+			}
+
+			if routerCreateOpts.AuthMode == "internal" && len(routerCreateOpts.User) > 0 && len(routerCreateOpts.Password) > 0 && !routerCreateOpts.EnableConsole {
```

Comment:
I am assuming that if a user wants to set up the user/pass values for the console, the console should be enabled as well. Do you agree @ajssmith ?